frontend/src/services/api/user.ts

```
@@ -1,27 +1,35 @@
 	return axios.delete(`/users/${id}`);
 };
 
-export const updateUserAPI = ({ id, ...rest }: IUserUpdatePayload) => {
-	return axios.put(`/users/${id}`, rest);
+export const updateUserAPI = (id: string, payload: IUserUpdatePayload) => {
+	return axios.patch(`/users/${id}`, payload);
 };
 
 export const getOnlineUsersAPI = () => {
 	return axios.get<IBackendRes<IOnlineUser>>('/users/online');
+	return axios.get<IBackendRes<IOnlineUser>>('/users/online');
```

Biome (2.1.2)

[error] This code will never be reached ...

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)